docs/application/vscode-ext/web.md

```diff
@@ -240,11 +328,62 @@ To run project:
 To debug application, follow these steps:
 
 1. In VS Code, click **View** > **Command Palette**.
-2. In the input field of the **Command Palette** that appears, type **Run Web Simulator** and press **Enter**.
+2. In the input field of the **Command Palette** that appears, type **Tizen Web: Run Web Simulator** and press **Enter**.
 3. After the **Web Simulator** is launched, enter the path of the file in the box. For example, file:///C://Workspace/basic/index.html.
 
     ![Debugging you application in the Web Simulator](media/tizen_web18.png)
 
+### Run Design Editor
+
+To run design editor, follow these steps:
+
+> **Note** 
+> 
+> **You must select the HTML extension file in the file explorer.**
+
+To run Design Editor, follow these steps:
+
+You can launch Design Editor in one of the following ways:
+- Command Palette
+- Design-Editor button
+- Right-click preference
+
+**Command Palette**
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Tizen Web: Run Design-Editor** and press **Enter**.
+
+    A new tab is created and the Design Editor is launched.
+
+**Design-Editor button**
+1. To launch the Design Editor, click Design-Editor at the bottom right of the screen.
+
+    ![Design Editor button](media/tizen_web27.png)
+
+**Rignt-click preference**
+1. Select the HTML file that you want to launch and right-click.
+2. From the list that appears, click **Tizen Web: Run Design-Editor**.
+
+    ![Design Editor right click](media/tizen_web28.png)
+
+Design Editor is successfully launched.
+
+![Design Editor launch](media/tizen_web29.png)
+
+### Quick Action Execution
+Right-click on the project folder to use the extension function in the menu.
+
+For more information on each function, see:
+- [Tizen Web: Build Tizen App](#Build-Project)
+- [Tizen Web: Debug Tizen App](#Deploy-and-Run-Application-in-Emulator)
+- [Tizen Web: Install Tizen App](#Deploy-and-Run-Application-in-Emulator)
+- [Tizen Web: Make Exclude Build List](#Exclude-Build-List)
+- [Tizen Web: Run Tizen App](#Deploy-and-Run-Application-in-Emulator)
+
+![Design Editor launch](media/tizen_web34.png)
+> **Note**
+>
+> To use the Quick action, right-click on the project folder.
```

Comment:
Question. This note seems containing same context with the describing line.
Is there any reason you emphasized it again in note form?

Reply:
Let's delete the note and highlight the project folder in the description.

done.